src/data/polynomial/eval.lean

```diff
@@ -818,6 +818,18 @@ lemma eval_prod {ι : Type*} (s : finset ι) (p : ι → R[X]) (x : R) :
   eval x (∏ j in s, p j) = ∏ j in s, eval x (p j) :=
 (eval_ring_hom x).map_prod _ _
 
+lemma list_prod_comp (l : list R[X]) (q : R[X]) :
+  l.prod.comp q = (l.map (λ p : R[X], p.comp q)).prod :=
+map_list_prod (comp_ring_hom q) _
+
+lemma multiset_prod_comp (s : multiset R[X]) (q : R[X]) :
```

Comment:
```suggestion
lemma multiset.prod_comp (s : multiset R[X]) (q : R[X]) :
```

Reply:
Maybe we can't do that.

If we use `multiset.prod_comp`, we would have to use `open _root_.multiset` instead of `open multiset` in `namespace polynomial`.

Reply:
I was about to make the same suggestion. The point is allowing dot notation.

Reply:
But these lemmas are in `namespace polynomial` and using dot notation will create a namespace in `namespace polynomial`? Or maybe we can use `_root_.multiset.prod_comp`?